### ISSUE TITLE
"webpack is not a function" when 'yarn start'

### ISSUE THREAD

**Issue by peterbe** [mozilla/delivery-console#834]
Ever since the upgrade from [react-dev-utils 7.0.5 to 8.0.0](https://github.com/mozilla/delivery-console/pull/823/files) `yarn start` stopped working: https://irccloud.mozilla.com/pastebin/b8q2yd77/

I have verified, on `master` that this works:

```bash
▶ yarn add react-dev-utils@7.0.5
▶ yarn start
```

**Comment by peterbe** [comment]
By the way, *with* `react-dev-utils@8.0.0` installed you can run `yarn run build` and `yarn test`. Obviously, since the [CI passed in the PR](https://github.com/mozilla/delivery-console/pull/823).

**Comment by peterbe** [comment]
`react-dev-utils` doesn't have its won CHANGELOG but the [7.0.5 to 8.0.0 diff is here](https://github.com/facebook/create-react-app/compare/react-dev-utils@7.0.5...react-dev-utils@8.0.0). 

If you look at [packages/react-dev-utils/package.json](https://github.com/facebook/create-react-app/compare/react-dev-utils@7.0.5...react-dev-utils@8.0.0#diff-40459d9699abb4c15a99b983904adba4) you can see that, apart from upping the min. node version, there's a new sub-package dependency called `fork-ts-checker-webpack-plugin@1.0.0-alpha.6` which I know nothing about other than the version string contains the word "alpha".

**Comment by peterbe** [comment]
Also, the file [`packages/create-react-app/yarn.lock.cached`](https://github.com/facebook/create-react-app/compare/react-dev-utils@7.0.5...react-dev-utils@8.0.0#diff-9cf5199c70b9fec6307377b9f374fcfa) now seems to include a change of:
`react-dev-utils@^7.0.4` to `react-dev-utils@^7.0.5` which is just confusing.

**Comment by peterbe** [comment]
It could easily be that we're suffering just because we ejected. If `yarn start` broke everyones' `create-react-app` apps we'd see this loud and clear in [the main issue tracker](https://github.com/facebook/create-react-app/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) but I don't see it. 

If we do revert our `react-dev-utils` version we're just going to forget why we had to do that and then some Renovate PR will upgrade it again and we'll forget and break ourselves again. Unless, there's some neat way (like with pyup pin comments) to prevent it and leave it like that till the heat death of the sun.
